packages/node_modules/cerebral/test/setup.js

```diff
@@ -1,5 +1,24 @@
 'use strict'
 
+require('core-js/es6')
+require('core-js/es7/reflect')
+require('zone.js/dist/zone-node')
+require('zone.js/dist/long-stack-trace-zone')
+require('zone.js/dist/proxy')
+require('zone.js/dist/sync-test')
+require('zone.js/dist/async-test')
+require('zone.js/dist/fake-async-test')
+require('zone.js/dist/proxy')
```

Comment:
zone.js patches a lot of async methods. Not sure I like the fact that all testing for cerebral is now tinted by this.

If we need such deep customisation of the testing environment for Angular, I propose to simply create a new package for angular. The only change for the user is this:

```ts
import { connect } from '@cerebral/angular'
// instead of
import { connect } from 'cerebral/angular'
```

Maybe all views should live in their own packages. '@cerebral/react', '@cerebral/xxx'. @christianalfoni what do you think ?

Reply:
The views are now moved to "@cerebral/". 
So Angular should also move there in this PR.

Reply:
Sounds good will work on that.

Reply:
I moved the angular views into @cerebral now